style.css

```diff
@@ -0,0 +1,245 @@
+*{
+	box-sizing: border-box; 
+}
+html{
+	font-size: 20px; 
+}
+body {
+	background-color:#771668;
+	width: 100%;
+	margin-top:80px;
+	margin-left:0px;
+	margin-right:0px;
+	margin-bottom:0px;
+}
+
+header{
+	position: fixed;
+	top:0px;
+	left:0px;
+	background-color: #2d094f;
+	width:100%;
+	padding: 10px;
+	border:3px solid black;
+	z-index:2;
+}
+
+.nav {
+	font-size: 1.5rem;
+	color: white;
+	display: flex;
+	gap:40px;
+	text-decoration: none;
+			
+}
+section.intro{
+		display: flex;
+		align-items: center;
+		flex-direction: column;
+		width: 100%;
+		margin-bottom:250px;
+		gap: 18px;			
+}
+
+p.intro{
+	color: white;
+	font-size: 6rem;
+	display: flex;
+	justify-content: center;
+	flex-wrap: wrap;
+	margin:0px;
+	align-items: center;
+	gap:0px;
+}
+
+p.little-intro{
+	color: white;
+	font-size: 2rem;
+	display: flex;
+	justify-content: center;
+}
+
+div.about-catcher{
```

Comment:
here you should select it without saying div, because that way you can later change your mind about what tag it is, e.g. change it to section

Reply:
same applies in lots of places - everywhere where you have used both the tag and the class, use only the class if possible